scripts/README.md

```diff
@@ -16,29 +16,34 @@ scripts/
 ```
 
 # initializeEs.js
+
 This script will initialize your ElasticSearch environment. It should be run during your initial setup of the project.
 
 > From the project root (`portal/`) or from this directory (`portal/scripts/`), run the following command. Note that you may need to change the ENV value used to match the environment declared in the `cms/pm2.config.json` file setup.
 
 ```
-ENV=prd npm run initializeEs
+ENV=env_prd npm run initializeEs
 ```
 
 You can specify the following environment variables when running this script:
-* **`ENV`: PM2 environment (i.e. `dev`, `prd`, etc.) <sup>*required*</sup>**
-* `CMS_CONFIG`: path to your CMS `pm2.config.js` file <sup>*optional*</sup>
-* `ES_INDEX`: models index name <sup>*optional*</sup>
-* `PROJECT_ID`: Arranger project id <sup>*optional*</sup>
-* `ES_HOST`: ElasticSearch host URL <sup>*optional*</sup>
+
+- **`ENV`: PM2 environment (i.e. `env_dev`, `env_prd`, etc.) <sup>_required_</sup>**
```

Comment:
These changes are just Prettier auto-formatting but I've updated the ENV examples to use real PM2 values